src/main/java/org/shanerx/tradeshop/commands/CommandRunner.java

```diff
@@ -564,7 +564,7 @@ private Shop findShop() {
 				return Shop.loadShop((Sign) b.getState());
 
 			} else if (plugin.getListManager().isInventory(b.getType()) &&
-					((InventoryHolder) b.getState()).getInventory().getName().contains("$ ^Sign:l_")) {
+                    ((InventoryHolder) b.getBlockData()).getInventory().getType().getDefaultTitle().contains("$ ^Sign:l_")) {
```

Comment:
Im rewriting this myself already, however just felt like letting you know that Inventory#getType()#getDefaultTitle() is probably returning the type name and not the custom name of the inventory. So if it's a chest named BOB it would return CHEST not BOB.

Reply:
Absolutely, I kinda just put whatever seemed close in there to make it build donIncould test with 1.14.1. I am not that great at making plugins. My main forte is PHP.